sycl/include/sycl/stl_wrappers/cmath-fallback.h

```diff
@@ -0,0 +1,337 @@
+#ifndef __CMATH_FALLBACK_H__
```

Comment:
`sycl/include/sycl/stl_wrappers/cmath` and `sycl/include/sycl/stl_wrappers/cmath-fallback.h` lack copyright header.